libraries/AP_HAL_ChibiOS/hwdef/CubeYellow/hwdef.dat

```diff
@@ -110,7 +110,7 @@ I2C_ORDER I2C2 I2C1
 # 6) SERIAL5: extra UART (usually RTOS debug console)
 
 # order of UARTs (and USB)
-UART_ORDER OTG1 UART4 USART2 USART3 UART8 UART7
+UART_ORDER OTG1 UART4 USART2 USART3 UART8 UART7 OTG2
```

Comment:
do we need to provide a define of the index into the serial drivers so that if something like SLCAN wants to find the 2nd USB port it can do so?

Reply:
We can default the last serial port to be SLCAN by default in SerialManager.

Reply:
no, don't just use the last one, the parser knows the index, so set the protocol default by that index.